packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -147,6 +148,12 @@ export class FeatureDevController {
                 case FollowUpTypes.SendFeedback:
                     this.sendFeedback()
                     break
+                case FollowUpTypes.AcceptAutoBuild:
+                    return this.processAutoBuildSetting(true, data)
+                case FollowUpTypes.DenyAutoBuild:
+                    return this.processAutoBuildSetting(false, data)
+                case FollowUpTypes.GenerateDevFile:
+                    return this.newTask(data, true)
```

Comment:
Can you create another function for this case?

Reply:
I will change the function to optionally accept "prefilledPrompt" instead